tests/Unit/SDK/Common/Attribute/AttributesTest.php

```diff
@@ -159,6 +159,17 @@ public function test_limit_get_total_added_values(): void
         $this->assertEquals(3, $attributes->getTotalAddedValues());
     }
 
+    public function test_limit_get_total_added_values_same_key(): void
```

Comment:
This could have a more descriptive name... `test_replace_attribute_does_not_increase_count` ?